content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,434 @@
+
+## Highlights of features graduating to Beta
+
+_This is a selection of some of the improvements that are now beta following the v1.32 release._
+
+### Job API managed-by mechanism
+
+The `managedBy` field for Jobs and was promoted to beta in the v1.32 release. This feature enables external controllers
+(like [Kueue](https://kueue.sigs.k8s.io/)) to manage Job synchronization, offering greater flexibility and integration
+with advanced workload management systems.
+
+This work was done as a part of [KEP #4368](https://github.com/kubernetes/enhancements/issues/4368), by [SIG
+Apps](https://github.com/kubernetes/community/tree/master/sig-apps).
+
+### Only allow anonymous auth for configured endpoints
+
+This feature lets admins specify which endpoints are allowed for anonymous requests. For example, the admin
+can choose to only allow anonymous access to health endpoints like `/healthz`, `/livez`, and `/readyz` while
+making sure preventing anonymous access to other cluster endpoints or resources even if a user
+misconfigures RBAC.
+    
+This work was done as a part of [KEP #4633](https://github.com/kubernetes/enhancements/issues/4633), by [SIG
+Auth](https://github.com/kubernetes/community/tree/master/sig-auth).
+
+### Per-plugin callback functions for accurate requeueing in kube-scheduler enhancements
+
+This feature enhance sthe scheduling throughput with more efficient scheduling retry decisions by
+per-plugin callback functions (QueueingHint). All plugins now have QueueingHints.
+
+This work was done as a part of [KEP #4247](https://github.com/kubernetes/enhancements/issues/4247), by [SIG
+Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling).
+
+### Recover from volume expansion failure
+
+This feature lets users recover from volume expansion failure by retrying with a smaller size. This enhancement ensures
+that volume expansion is more resilient and reliable, reducing the risk of data loss or corruption during the process.
+
+This work was done as a part of [KEP #1790](https://github.com/kubernetes/enhancements/issues/1790), by [SIG
+Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+### Volume group snapshot
+
+This feature introduces a VolumeGroupSnapshot API.
+Allows users to take a snapshot of multiple volumes together, ensuring data consistency across the volumes.
+
+This work was done as a part of [KEP #3476](https://github.com/kubernetes/enhancements/issues/3476), by [SIG
+Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+### Structured parameter support
+
+The core part of Dynamic Resource Allocation (DRA), the structured parameter support, got promoted to beta.
+This allows the kube-scheduler and Cluster Autoscaler to simulate claim allocation directly, without needing a
+third-party driver.
+Maning they can predict whether resource requests can be fulfilled based on the cluster's current state without actually
```

Comment:
```suggestion
These components can now predict whether resource requests can be fulfilled based on the cluster's current state without actually
```

Reply:
Fixed and accepted suggestion (it was meant to be "meaning" ofc)